corehq/apps/geospatial/reports.py

```diff
@@ -277,6 +278,31 @@ def _aggregate_query(self, query):
             precision = find_precision(query, case_property)
         return apply_geohash_agg(query, case_property, precision)
 
+    @property
+    def features_from_request(self):
+        features = {}
+        features_json = self.request.GET.get('features')
+        if features_json:
+            try:
+                features |= json.loads(features_json)
```

Comment:
nit: just curious about the usage of `|=` here. Just the usual `=` won't do?

Reply:
Ah, you are correct in that we can simply use `=`. This was old code from when we were still adding both the features and selected polygon to the same dict. I'll fix this.

Reply:
[253501a](https://github.com/dimagi/commcare-hq/pull/34511/commits/253501a18f7586e154cb143acfbc60891af3b747)